### PR TITLE
flags: fix "message" string handling

### PR DESCRIPTION
**Pull request by bmeneguele**
Solves issue #376 wrt string handling in `--message | -m` flags.

### CODE REVIEW

**Comment by prarit** [comment]
@bmeneguele 

This looks good, however, I think adding 'message' tests for the commands that explicitly test message input like "{\"alpha\": \"beta\"}" is a good idea.  After all, that is the bug you're fixing.  Take a look at cmd/mr_note_test.go:Test_mrNoteMsg() for example.  You could easily add a new test to that file that tests the example input.

**Comment by bmeneguele** [comment]
Yes, of course. It makes huge sense in adding a test for it.
Thanks for the heads up @prarit  :P

**Review by zaquestion — APPROVED** [review]
Looks great, not that adding a test is a bad move either ;) 

@bmeneguele I've sent an invite to add you as a collaborator as well :D Thanks for all the great work to improve lab.

**Comment by bmeneguele** [comment]
> @bmeneguele I've sent an invite to add you as a collaborator as well :D Thanks for all the great work to improve lab.

I'm glad I can help! 
Tyvm @zaquestion

**Comment by bmeneguele** [comment]
I added the test for the string causing issues, but it had to follow a different path from what you suggested @prarit. After checking the test source I noticed that it was testing the resulting note msg being created, giving no room for checking the msg coming from the argument list (during app exec).

The issue happens right in the argument list parsing, so I added the test by calling the 'lab' with the problematic text and checked its output. What do you think?

File: cmd/mr_note_test.go [hunk 1]
```diff
@@ -23,6 +23,19 @@ func Test_mrCreateNote(t *testing.T) {
...
+	cmd = exec.Command(labBinaryPath, "mr", "note", "lab-testing", "1",
+		"-m", "{\"key\": \"value\"}")
+
+	b, err = cmd.CombinedOutput()
+	if err != nil {
+		t.Log(string(b))
+		t.Fatal(err)
+	}
+
+	require.Contains(t, string(b), "https://gitlab.com/lab-testing/test/merge_requests/1#note_")
```
Comment (zaquestion) [inline cmd/mr_note_test.go:38]: Suggest wrapping this in a subtest, and linking to the issue ( https://github.com/zaquestion/lab/issues/376 )

Comment (bmeneguele) [reply cmd/mr_note_test.go:38]: Sure thing. Will do.
Thanks!

**Comment by bmeneguele** [comment]
@zaquestion I finally added the subtest for the problematic string and also linked it to the issue number.
Sorry for vanishing for some days :)

**Pull request merged**